mac/parser.py

```diff
@@ -36,7 +50,7 @@ def parse(companies):
     # We only care about the first line since it contains the data we need
     for company in companies_list:
         company_lines = company.split("\n")
-        mac_name = company_lines[0].split("   (hex)		")
-        to_return[mac_name[0]] = mac_name[1].replace("\r", "")
+        mac, name = company_lines[0].split("   (hex)		")
+        to_return[mac] = name.rstrip("\r")
```

Comment:
Interesting way to simplify this code, I didn't know this was possible in python

Reply:
yea, I tried to implement dataframes and i tried to make it go from Dict[str, str] to Dict[str, List[str]] and then when I went to change it back it ended up like this lol